### ISSUE TITLE
Learning rate is reinitialized after continuing training with adversarial_lr option

### ISSUE THREAD

**Issue by 1027663760** [AlexeyAB/darknet#8496]
x_size = 41336832, original_delta = 0000000DA1C00000, original_input = 0000000D97E00000, net.learning_rate = 0.694440
During training I observed that net.learning_rate is getting bigger and bigger, when I stop training, use below command to continue training
detector train Data\voc.data cfg\yolov4-tiny_3l_spp_mish.cfg Data\backup\yolov4-tiny_3l_spp_mish_last.weights -i 0
net.learning_rate is reinitialized to a very low value
Is this a normal result? Or darknet forgot to count the number of iterations in this part
@AlexeyAB
